PendingReleaseNotes.md

```diff
@@ -4,3 +4,4 @@
 
 
 ## Features
+* [Add support for rbd mirroring for rbd block pool rados-namesapce](https://github.com/rook/rook/pull/14701).
```

Comment:
Let's not add features to the 1.16 pending release notes, if planning on backporting to 1.15.

Reply:
Maybe it will take time for the whole feature to get ready with monitoring and CI, 
we will just backport the enable-disable for now, and can call out with 1.16

Reply:
If we want to backport this, we need to revert this file or else it will cause a merge conflict during the 1.15 backport.